packaging/centos7/cloud.spec

```diff
@@ -58,7 +58,7 @@ intelligent IaaS cloud implementation.
 
 %package management
 Summary:   CloudStack management server UI
-Requires: java => 1.8.0
```

Comment:
Sorry if this is a silly question, but the `requires` notation needs a full package name? Is that why you needed to change it?

Reply:
no, the jdk is need and not just the jre

Reply:
Ah sure. 
Would it be possible to use `Requires: javac >= 1.8.0`?

Reply:
will look into that

Reply:
No, only jre is needed but on centos java-1.8.0-openjdk is the jre pkg and java-1.8.0-openjdk-devel has the jdk.